cfpb/cfgov-refresh
# Updating copy on youtube video

Updating copy on youtube video


## Testing

- Create a live event
- Visit the event and click the `YouTube troubleshooting guide` link.
- You should navigate to `https://support.google.com/youtube/answer/3037019?hl=en&co=GENIE.Platform%3DDesktop&oco=1`.


## Screenshots
- 
<img width="300" alt="screen shot 2017-04-20 at 5 11 48 pm" src="https://cloud.githubusercontent.com/assets/1696212/25252815/7bf61208-25ec-11e7-8d0d-c5ca7a5fb4c2.png">


## Checklist
